build.gradle.kts

```diff
@@ -49,7 +49,6 @@ dependencies {
   testImplementation("io.github.http-builder-ng:http-builder-ng-apache:1.0.4")
   testImplementation("org.wiremock:wiremock-standalone:3.9.1")
   testImplementation("junit:junit:4.13.2")
-  testImplementation("net.javacrumbs.json-unit:json-unit-assertj:3.4.1")
   testImplementation("org.awaitility:awaitility:4.2.2")
```

Comment:
It wasn't the intention to replace this but turns out just a few helper functions is more readable than the JSONAssert library